src/plugins/wizard/server/capabilities_provider.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const capabilitiesProvider = () => ({
+  wizard: {
+    // TODO: investigate which capabilities we need to provide
+    // createNew: true,
+    // createShortUrl: true,
+    // delete: true,
+    show: true,
+    // showWriteControls: true,
+    // save: true,
+    // saveQuery: true,
+  },
```

Comment:
I'll defer this to you, since you have more context here. Not entirely sure what the capabilities provider does w.r.t saved object management.

Reply:
Yeah, I’m not at all an expert in how capabilities work generally… but given the way the user flow works for D&D (we start by creating a visualization, and the result is essentially a visualization), using the existing visualization plugin capabilities actually makes more sense. So instead of defining and checking `wizard.show` or `wizard.save`, it seems like it makes more sense to just check for `visualization.show` or `visualization.save` capabilities (which is what I do in the embeddable to see whether to enable the `Edit wizard` action. Because we don’t really have a use case where a user has wizard capabilities that are different from their visualization capabilities.

But we'll definitely want to come back and figure out what exactly we should do here.